paasta_tools/cli/cmds/metastatus.py

```diff
@@ -59,15 +59,33 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    status_parser.add_argument(
+        '-g',
+        '--groupings',
+        nargs='+',
+        default=['region'],
+        help=(
+            'Group resource information of slaves grouped by attribute.'
+            'Note: This is only effective with -vv'
+        )
+    )
+    status_parser.add_argument('-H', '--humanize', action='store_true', dest="humanize", default=False,
```

Comment:
Is there any reason *not* to make it human readable by default and make our computers remember to pass in the false when they know they don't want human readable units?